src/pycolmap/scene/database_cache.cc

```diff
@@ -19,6 +19,8 @@ void BindDatabaseCache(py::module& m) {
                   "min_num_matches"_a,
                   "ignore_watermarks"_a,
                   "image_names"_a)
+      .def("add_camera", &DatabaseCache::AddCamera)
+      .def("add_image", &DatabaseCache::AddImage)
```

Comment:
AFAIK `DatabaseCache` does not have such methods?

Reply:
Right. Not done here. I am now expanding the scope of this PR.